src/frontend/css/modules/header.less

```diff
@@ -59,6 +59,13 @@
       .drop-menu-open-animation(top right);
     }
 
+    li.hover-scale-animation:hover {
```

Comment:
You should specify scope, otherwise, it will apply for all system

Reply:
it is defined in `#header`, what more do we need?

Reply:
Ok. I see.